src/cve/stages/convert_to_output_object.py

```diff
@@ -73,11 +74,16 @@ def _parse_agent_morpheus_engine_output(row: dict) -> AgentMorpheusEngineOutput:
     justification_output = JustificationOutput(label=row['justification_label'],
                                                reason=row['justification'],
                                                status=row['affected_status'])
-
+    
+    # Combine CVSS model outputs into a single CVSSOutput object
+    cvss_output = CVSSOutput(vector_string=row['vector_string'],
+                             score=row['score'])
+    
```

Comment:
Please change col names literals to already created constants.
And in addition, only if decided to go with making the feature conditional on a config setting, add the following If Else instead:
```suggestion
    # Combine CVSS model outputs into a single CVSSOutput object
    if row.get(CVSS_VECTOR_STRING_COL_NAME, None) and 
       row.get(CVECVSSNode.CVSS_SCORE_COL_NAME, None) :
        cvss_output = CVSSOutput(vector_string=row['vector_string'],
                                                         score=row['score'])
    else:
        cvss_output=None
```